ansible/roles/base/tasks/main.yml

```diff
@@ -17,19 +15,24 @@
 
     - name: Install Python 3.6
       apt:
-        name: python3.6
-        state: latest
+       name: python3.6
+       state: latest
 
     - name: install python-pip
-      apt: name=python-pip
+      apt: 
+       name: python3-pip
       become: yes
+      become_method: sudo
 
-    - name: install python-virtualenv
-      apt: name=python-virtualenv
+    - name: Install python-virtualenv
+      apt:
+       name: python-virtualenv
       become: yes
 
     - name: Install PIP requirements
+      become: yes
+      become_method: sudo -H
       pip:
         requirements: /vagrant/ansible/pip-requirements.txt
-        virtualenv: /home/vagrant/python3.6
+        virtualenv: /vagrant/python3.6
```

Comment:
Also, I see below, now all the files are in /python3.6/… I don’t think that’s what we want… That means when/if we ever change to 3.7 now we’re going to have a separate folder, etc. I’m not sure we want that. The root code should just be in /src/ or something like that.